src/Bundle/Command/DebugGridCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Command;
+
+use Sylius\Component\Grid\Provider\GridProviderInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Dumper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+#[AsCommand(name: 'sylius:debug:grid')]
```

Comment:
```suggestion
#[AsCommand(name: 'sylius:debug:grid', description: 'Debug grid configuration')]
```

<img width="1710" height="107" alt="image" src="https://github.com/user-attachments/assets/56f6c5b5-5c51-4c1b-94e5-1c40156ab80b" />